scripts/test.py

```diff
@@ -10,6 +10,11 @@
 MACOS = sys.platform == 'darwin'
 
 assert 'EM_CONFIG' in os.environ, "emsdk should be activated before running this script"
+# Remove the EM_CACHE environment variable.  It interferes with testing since 
+# it would otherwise be fixed for the duration of the script and we expect
+# "emsdk activate" to be able swtich between SDKs during the running of this
+# script.
+del os.environ['EM_CACHE']
```

Comment:
doesn't `activate` change the value of that env var, so it would switch properly? or is that set from somewhere else?

Reply:
Its it not possible for a child process to modify the environment of the parent process.  So the is no way activate can do this.  This is why we have the `source ./emsdk_env.sh` thing... the source in bash takes case or modifying the running shell.

Within this python script the environment cannot be changed by calling a subprocess.

Reply:
I see, thanks.